modules/build/src/main/scala/scala/build/preprocessing/directives/DirectiveHandler.scala

```diff
@@ -10,4 +13,11 @@ trait DirectiveHandler {
 
   // Strict / using_directives-based directives
   def keys: Seq[String]
+
+  def handleValues(
+    directive: StrictDirective,
+    path: Either[String, os.Path],
+    cwd: ScopePath
+  ): Either[BuildException, (Option[T], Seq[Scoped[T]])]
```

Comment:
Why do we abstract over `T` here?

Reply:
I've made here the abstraction to be able to reuse DirectiveProcessor.process for require and using directives